_posts/2020-03-15-method-s-arguments-pt-1.markdown

```diff
@@ -0,0 +1,372 @@
+---
+layout: post
+title: Ruby Method's Arguments
```

Comment:
@arielj what do you think if we change the title to "Exploring Method Arguments in Ruby: Part 1"?

Reply:
Yeah sure, it's more catchy, I'll change it tomorrow

Reply:
@arielj I can change it, no worries